agent/agent.go

```diff
@@ -245,21 +262,26 @@ func (a *Agent) fetchProjectConfig(ctx context.Context, tc *taskContext) error {
 	return nil
 }
 
-func (a *Agent) resetLogging(ctx context.Context, tc *taskContext) {
+func (a *Agent) resetLogging(ctx context.Context, tc *taskContext) error {
+	var err error
 	if tc.logger != nil {
 		grip.Error(tc.logger.Close())
 	}
 	grip.Error(os.RemoveAll(filepath.Join(a.opts.WorkingDirectory, taskLogDirectory)))
 	if tc.project != nil && tc.project.Loggers != nil {
-		tc.logger = a.makeLoggerProducer(ctx, tc, tc.project.Loggers, "")
+		tc.logger, err = a.makeLoggerProducer(ctx, tc, tc.project.Loggers, "")
 	} else {
-		tc.logger = a.comm.GetLoggerProducer(ctx, tc.task, nil)
+		tc.logger, err = a.comm.GetLoggerProducer(ctx, tc.task, nil)
+	}
+	if err != nil {
+		return err
```

Comment:
wrap?

Reply:
I believe this to be sufficiently wrapped already